include/odp/api/spec/schedule_types.h

```diff
@@ -194,6 +261,16 @@ typedef	struct odp_schedule_param_t {
 	  *
 	  * Default value is 0. */
 	uint32_t lock_count;
+
+	/** Queue specific cache stashing hints
+	  *
+	  * Depending on the implementation, configuring these may improve
+	  * performance. Overrides any group or priority specific configuration
+	  * for this queue. By default, all entries are disabled
+	  * (see odp_cache_stash_region_t::l2::enable and
+	  * odp_cache_stash_region_t::l3::enable). */
+	odp_cache_stash_config_t cache_stash_hints;
```

Comment:
Another place for this could be queue param struct (one level up). In this PR, we could still say that it's ignored for plain queues, but it would be then easier to enable stashing for plain queues later (stashing could happen in dequeue time).